lib/volk_malloc.c

```diff
@@ -50,6 +50,12 @@
 
 void* volk_malloc(size_t size, size_t alignment)
 {
+    // The ASAN address sanitizer requires the allocation size to be a multiple of
+    // the the alignment. Add bytes to size so that it is a multiple of alignment
+    // if it's not.
```

Comment:
Recommend something more like the following for the comment text:
```
Regardless of the changes here to make ASAN (address sanitizer) checking happy, calling `volk_malloc` might result in the allocation of more memory than requested for correct alignment. Any allocation size change here will in general not impact the end result, since initial size alignment is required either way.
```